war/src/main/less/modules/panes-and-bigtable.less

```diff
@@ -0,0 +1,142 @@
+  background: var(--pane-header-bg);
+  // TODO: see how this affects the side panel widgets
+  border: var(--pane-border-width) solid var(--pane-header-border-color);
+  border-left: none;
+  border-right: none;
+  color: var(--pane-text-color);
+}
+
+.pane-header {
+  border-top: none;
+  border-top-left-radius: 4px;
+  border-top-right-radius: 4px;
+}
+
+.pane-footer {
+  border-bottom: none;
+  border-bottom-left-radius: 4px;
+  border-bottom-right-radius: 4px;
+}
+
+.pane td {
+  padding: 0.25rem;
+  vertical-align: middle;
+}
+
+table.pane {
+  width: 100%;
+  border-collapse: collapse;
+  border: var(--pane-border-width) solid var(--pane-border-color);
+}
+
+td.pane {
+  padding: 0.25rem;
+  vertical-align: middle;
+}
+
+table.stripped tr:nth-of-type(even) {
+  background: var(--even-row-color);
+}
+
+table.stripped-even tr:nth-child(even) {
+  background: var(--even-row-color);
+}
+table.stripped-odd tr:nth-child(odd) {
+  background: var(--even-row-color);
+}
+table.stripped tr:hover, table.stripped-even tr:hover, table.stripped-odd tr:hover {
+  background: var(--table-striped-bg--hover) !important;
+}
+
+div.pane-header {
+  font-weight: var(--pane-header-font-weight);
+  padding-right: 24px;
+}
+
+div.pane-header .collapse {
+  float: right;
+  margin-left: 3px;
+}
+
+th.pane {
+  font-weight: var(--pane-header-font-weight);
+}
+
+/* Bigtable */
+
+table.bigtable {
+  border: none; // Needs to be set AFTER the table.pane declaration
+}
+
+/* For non-full screen table */
+
+.bigtable.width-auto {
+  width: auto;
+}
+
+// Keep this for tables still wrapped by .pane-frame
+.pane-frame table, .pane-frame .bigtable tr {
+  border: none; /* Border will be provided by the pane-frame */
+}
+
+.bigtable th {
+  font-size: var(--bigtable-header-font-size); // TODO: move this to a <thead> or .header declaration
+  font-weight: var(--bigtable-header-font-weight);
+  color: var(--bigtable-header-text-color);
+  background: var(--bigtable-header-bg);
+  white-space: nowrap;
+  border-top: var(--bigtable-border-width) solid var(--bigtable-header-bg);
+  text-align: left;
+
+  &.minimum-width {
+    width: 1px;
+  }
+
+  &[align=right] {
+    text-align: right;
+  }
+}
+
+.bigtable td {
+  border-top: var(--bigtable-border-width) solid var(--bigtable-row-border-color);
+  vertical-align: middle;
+}
+.bigtable tr:last-child td {
+  border-bottom: var(--bigtable-border-width) solid var(--bigtable-row-border-color);
+}
+
+.bigtable td,
+.bigtable th {
+  padding: var(--bigtable-cell-padding-y) var(--bigtable-cell-padding-x);
+}
+
+.bigtable .pane-header,
+.bigtable .pane-footer {
+  border-radius: 0;
+}
+
+/* ========================= sortable table ========================= */
+table.sortable a.sortheader {
+  text-decoration: none;
+  display: block;
+  // color: var(--pane-text-color);
+  color: var(--bigtable-header-text-color);
+  font-weight: var(--pane-header-font-weight);  // Should we use the bigtable one here?
```

Comment:
have you decided which one should be used?

Reply:
I have decided on using both. I'll use `--pane-header-color` on `table.sortable` by default but will override it for `bigtable`s. (See https://github.com/jenkinsci/jenkins/pull/4835/commits/4527d84056bd503e226e0f450d2ba6da536cff71)